src/_guides/language/sound-dart.md

```diff
@@ -1,54 +1,51 @@
 ---
-title: Strong Mode Dart
+title: Dart's Type System
```

Comment:
I think we wanted to move toward this being a page about the type system, in general. Otherwise, I might've gone back to its first name, "Sound Dart".

Opinions, @anders-sandholm / @mit-mit / @mjohnsullivan?

Reply:
I really like the title 'Dart's Type System' and I think it's a nice general title for a page that covers an overall overview of how the typing system works.